### PR TITLE
Add TEM callouts

### PR DESCRIPTION
**Pull request by estherk15**
### What does this PR do? What is the motivation?

- Add a callout for the TEM enablement webinar sessions
- https://datadoghq.atlassian.net/browse/DOCS-8195

### Merge instructions

**Pending TEM team approval**

### CODE REVIEW

File: content/en/getting_started/dashboards/_index.md [hunk 1]
```diff
@@ -24,6 +24,10 @@ further_reading:
     text: "Join an interactive session on better visualizations with Dashboards"  
 ---
 
+{{< learning-center-callout header="Join an enablement webinar session" btn_title="Sign Up" btn_url="https://www.datadoghq.com/technical-enablement/session/datadog-overview/">}}
```
Comment (estherk15) [inline content/en/getting_started/dashboards/_index.md:27]:
```suggestion
{{< learning-center-callout header="Join an enablement webinar session" btn_title="Sign Up" btn_url="https://www.datadoghq.com/technical-enablement/sessions/?tags.topics-0=Dashboarding">}}
```

File: content/en/cloud_cost_management/_index.md [hunk 2]
```diff
@@ -33,6 +33,11 @@ cascade:
 <div class="alert alert-warning">Cloud Cost Management is not supported for your selected <a href="/getting_started/site">Datadog site</a> ({{< region-param key="dd_site_name" >}}).</div>
 {{< /site-region >}}
 
+
+{{< learning-center-callout header="Join an enablement webinar session" btn_title="Sign Up" btn_url="https://www.datadoghq.com/technical-enablement/sessions/?tags.topics-0=Cloud+Cost+Management">}}
+  Explore your cloud provider costs correlated with your real-time telemetry data to gain actionable insights and alerts on where your cloud costs are coming from, how they are changing, and where to find potential optimizations.
```
Comment (urseberry) [inline content/en/cloud_cost_management/_index.md:38]:
```suggestion
  Explore your cloud provider costs and correlate them with real-time telemetry data. Gain actionable insights and alerts on where your cloud costs are coming from, how they are changing, and where to find potential optimizations.
```

File: content/en/containers/kubernetes/_index.md [hunk 3]
```diff
@@ -26,6 +26,11 @@ further_reading:
       text: 'Join an interactive session for insights on Kubernetes monitoring'
 ---
 
+
+{{< learning-center-callout header="Join an enablement webinar session" btn_title="Sign Up" btn_url="https://www.datadoghq.com/technical-enablement/sessions/?tags.topics-0=Kubernetes">}}
+  This foundation enablement session will focus solely on how Datadog can monitor Kubernetes. Learn how to configure Datadog for Kubernetes and how to get started. Explore the various views and tools Datadog offers to visualize and analyze your cluster and application metrics, traces, and logs.
```
Comment (urseberry) [inline content/en/containers/kubernetes/_index.md:31]:
```suggestion
  This foundation enablement session will focus on how Datadog can monitor Kubernetes. Learn how to configure Datadog for Kubernetes and how to get started. Explore the various views and tools Datadog offers to visualize and analyze your cluster and application metrics, traces, and logs.
```

File: content/en/continuous_integration/_index.md [hunk 4]
```diff
@@ -42,6 +42,11 @@ cascade:
 
 <div class="alert alert-info">This page is about bringing your continuous integration (CI) metrics and data into Datadog dashboards. If you want to run Continuous Testing tests in your CI pipelines, see the <a href="/continuous_testing/cicd_integrations/" target="_blank">Continuous Testing and CI/CD</a> section.</div>
 
+{{< learning-center-callout header="Join an enablement webinar session" btn_title="Sign Up" btn_url="https://www.datadoghq.com/technical-enablement/sessions/?tags.topics-0=CI">}}
+  Join the Introduction to CI Visibility session to understand how Datadog CI Visibility enhances the efficiency of CI pipelines and where to configure the Testing Visibility and Pipeline Visibility products.
```
Comment (urseberry) [inline content/en/continuous_integration/_index.md:46]:
```suggestion
  Join the Introduction to CI Visibility session to understand how Datadog CI Visibility enhances the efficiency of CI pipelines and how to configure the Testing Visibility and Pipeline Visibility products. 
```

File: content/en/database_monitoring/_index.md [hunk 5]
```diff
@@ -23,6 +23,12 @@ cascade:
     algolia:
         rank: 70
 ---
+
+
+{{< learning-center-callout header="Join an enablement webinar session" btn_title="Sign Up" btn_url="https://www.datadoghq.com/technical-enablement/sessions/?tags.topics-0=Database">}}
+  Learn how with Database Monitoring, you can quickly pinpoint costly and slow queries and drill into precise execution details to address bottlenecks.
```
Comment (urseberry) [inline content/en/database_monitoring/_index.md:29]:
```suggestion
  With Database Monitoring, learn how to quickly pinpoint costly and slow queries. Drill into precise execution details to address bottlenecks.
```

File: content/en/getting_started/_index.md [hunk 6]
```diff
@@ -81,6 +81,10 @@ Additionally, hundreds of [integrations][1] allow you to layer Datadog features
 
 ## Learn more
 
+{{< learning-center-callout header="Join an enablement webinar session" btn_title="Sign Up" btn_url="https://www.datadoghq.com/technical-enablement/session/datadog-overview/">}}
+  This foundation enablement session will help answer the key question: 'What is Datadog, and what can it do for me?' You will learn how to send data to Datadog and which pages to visit to better understand the state of your various environments, applications, and infrastructure.
```
Comment (urseberry) [inline content/en/getting_started/_index.md:85]:
```suggestion
  This foundation enablement session helps answer the key question: "What is Datadog, and what can it do for me?" You will learn how to send data to Datadog and which pages to visit to better understand the state of your various environments, applications, and infrastructure.
```

File: content/en/getting_started/dashboards/_index.md [hunk 7]
```diff
@@ -24,6 +24,10 @@ further_reading:
     text: "Join an interactive session on better visualizations with Dashboards"  
 ---
 
+{{< learning-center-callout header="Join an enablement webinar session" btn_title="Sign Up" btn_url="https://www.datadoghq.com/technical-enablement/sessions/?tags.topics-0=Dashboarding">}}
+  Explore and register for Foundation Enablement sessions. Learn to customize your dashboards using our vast library of visualizations and easy-to-use drag-and-drop Dashboard builder. Enable your team's success by easily sharing data with stakeholders via reports, public URLs, and notebooks.
```
Comment (urseberry) [inline content/en/getting_started/dashboards/_index.md:28]:
```suggestion
  Explore and register for Foundation Enablement sessions. Learn to customize your dashboards using our library of visualizations and the drag-and-drop Dashboard builder. Enable your team's success by sharing data with stakeholders via reports, public URLs, and notebooks.
```

File: content/en/metrics/_index.md [hunk 8]
```diff
@@ -10,6 +10,11 @@ cascade:
         rank: 70
         tags: ["submit metrics", "metrics submission"]
 ---
+
+{{< learning-center-callout header="Join an enablement webinar session" btn_title="Sign Up" btn_url="https://www.datadoghq.com/technical-enablement/sessions/?tags.topics-0=Metrics">}}
+  Explore and register for Foundation Enablement sessions for Custom Metrics. Learn how custom metrics help you track your application KPIs, such as the number of visitors, average customer basket size, request latency, or performance distribution for a custom algorithm.
```
Comment (urseberry) [inline content/en/metrics/_index.md:15]:
```suggestion
  Explore and register for Foundation Enablement sessions for custom metrics. Learn how custom metrics help you track your application KPIs, such as the number of visitors, average customer basket size, request latency, or performance distribution for a custom algorithm.
```

File: content/en/metrics/custom_metrics.md [hunk 9]
```diff
@@ -29,6 +29,10 @@ algolia:
   tags: ['custom metrics']
 ---
 
+{{< learning-center-callout header="Join an enablement webinar session" btn_title="Sign Up" btn_url="https://www.datadoghq.com/technical-enablement/sessions/?tags.topics-0=Metrics">}}
+  Explore and register for Foundation Enablement sessions for Custom Metrics. Learn how custom metrics help you track your application KPIs, such as the number of visitors, average customer basket size, request latency, or performance distribution for a custom algorithm.
```
Comment (urseberry) [inline content/en/metrics/custom_metrics.md:33]:
```suggestion
  Explore and register for Foundation Enablement sessions for custom metrics. Learn how custom metrics help you track your application KPIs, such as the number of visitors, average customer basket size, request latency, or performance distribution for a custom algorithm.
```

**Comment by estherk15** [comment]
Thanks @urseberry!

**Review by urseberry — APPROVED** [review]

**Pull request merged**